apps/emqx_conf/src/emqx_conf_cli.erl

```diff
@@ -168,82 +249,109 @@ redact(Logs = #{cmd := license, args := [<<"update">>, _License]}) ->
+        true ->
+            emqx_ctl:warning("~ts~n", [Reason]);
+        false ->
+            emqx_ctl:warning("~p~n", [Reason])
+    end.
+
 usage_conf() ->
+    NamespaceOptLine = {"", "Set `--namespace <ns>` to narrow the scope to a specific namespace."},
     [
-        {"conf reload --replace|--merge", "reload etc/emqx.conf on local node"},
+        {"conf reload [--namespace <ns>] --replace|--merge", "reload etc/emqx.conf on local node"},
         {"", "The new configuration values will be overlaid on the existing values by default."},
         {"", "use the --replace flag to replace existing values with the new ones instead."},
+        NamespaceOptLine,
         {"----------------------------------", "------------"},
         {"conf show_keys", "print all the currently used configuration keys."},
-        {"conf show [<key>]",
+        {"----------------------------------", "------------"},
+        {"conf show [--namespace <ns>] [<key>]",
             "Print in-use configs (including default values) under the given key."},
         {"", "Print ALL keys if key is not provided"},
-        {"conf load --replace|--merge <path>", "Load a HOCON format config file."},
+        NamespaceOptLine,
+        {"----------------------------------", "------------"},
+        {"conf load [--namespace <ns>] --replace|--merge <path>",
+            "Load a HOCON format config file."},
         {"", "The new configuration values will be overlaid on the existing values by default."},
         {"", "use the --replace flag to replace existing values with the new ones instead."},
         {"", "The current node will initiate a cluster wide config change"},
         {"", "transaction to sync the changes to other nodes in the cluster. "},
         {"", "NOTE: do not make runtime config changes during rolling upgrade."},
+        NamespaceOptLine,
         {"----------------------------------", "------------"},
-        {"conf remove <conf-path>", "Removes a config path (e.g. `a.b.c`) from the current config."}
+        {"conf remove [--namespace <ns>] <conf-path>",
+            "Removes a config path (e.g. `a.b.c`) from the current config."},
+        NamespaceOptLine,
+        {"----------------------------------", "------------"}
     ].
 
 usage_sync() ->
+    NamespaceOptLine =
+        {"",
+            "Set `--namespace <ns>` to narrow the scope to a specific namespace"
+            " when inspecting configurations."},
     [
-        {"conf cluster_sync status", "Show cluster config sync status summary for all nodes."},
+        {"conf cluster_sync status [--namespace <ns>]",
+            "Show cluster config sync status summary for all nodes."},
+        NamespaceOptLine,
         {"conf cluster_sync inspect <ID>",
             "Inspect detailed information of the config change transaction at the given commit ID"},
-        {"conf cluster_sync skip [node]",
+        {"conf cluster_sync skip [--namespace <ns>] [node]",
```

Comment:
this cannot skip for a specific namespace, skip over a cluster-rpc transaction means skip for all namespaces.